deploy-board/deploy_board/templates/configs/new_capacity_adv.html

```diff
@@ -373,7 +373,8 @@ <h4 class="panel-title pull-left">Capacity</h4>
             var scope = this;
             var provider = capacitySetting.currentProvider;
             value = 'cmp_base-ebs-18.04';
-            if (this.selectedHostTypeValue.startsWith('EbsCompute')) {
+            const selectedHostTypeObject = capacityCreationInfo.hostTypes.get({"id":this.selectedHostTypeValue});
```

Comment:
why use get here and find at other places?

Reply:
never mind.

Reply:
Thank you for noticing that. I'm fixing this.